packages/flutter/lib/src/services/platform_views.dart

```diff
@@ -146,9 +312,29 @@ class AndroidViewController {
     });
   }
 
+  /// Sends an Android [MotionEvent](https://developer.android.com/reference/android/view/MotionEvent)
+  /// to the view.
+  ///
+  /// The Android MotionEvent object is created with [MotionEvent.obtain](https://developer.android.com/reference/android/view/MotionEvent.html#obtain(long,%20long,%20int,%20float,%20float,%20float,%20float,%20int,%20float,%20float,%20int,%20int)).
+  /// See documentation of [MotionEvent.obtain](https://developer.android.com/reference/android/view/MotionEvent.html#obtain(long,%20long,%20int,%20float,%20float,%20float,%20float,%20int,%20float,%20float,%20int,%20int))
+  /// for description of the parameters.
+  Future<void> sendMotionEvent(AndroidMotionEvent event) async {
+    await SystemChannels.platform_views.invokeMethod(
+        'touch',
+        <dynamic>[
+          id,
+        ]..addAll(event._asList())
```

Comment:
This seems more expensive then it needs to be - avoid the extra copying by passing in the list to asList?